File: nordeus_challenge_backend_spring_boot/src/main/java/com/mmileticc/nordeuschallenge/model/dto/RunHeroProgressDTO.java
package com.mmileticc.nordeuschallenge.model.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class RunHeroProgressDTO {

    private Long heroId;
    private String heroName;
    private int level;
    private int xp;
    private int xpToNextLevel;
    private int maxHealth;
    private int currentHealth;
    private int attack;
    private int defense;
    private int magic;
}

File: Gauntlet of the Hero - GOTH/Assets/_Scripts/Models/Backend/RunHeroProgressDto.cs
using Newtonsoft.Json;

namespace GauntletOfTheHero.Backend.Models
{
    [System.Serializable]
    public class RunHeroProgressDto
    {
        [JsonProperty("heroId")]
        public long heroId;

        [JsonProperty("heroName")]
        public string heroName;

        [JsonProperty("level")]
        public int level;

        [JsonProperty("xp")]
        public int xp;

        [JsonProperty("xpToNextLevel")]
        public int xpToNextLevel;

        [JsonProperty("maxHealth")]
        public int maxHealth;

        [JsonProperty("currentHealth")]
        public int currentHealth;

        [JsonProperty("attack")]
        public int attack;

        [JsonProperty("defense")]
        public int defense;

        [JsonProperty("magic")]
        public int magic;
    }
}